docs/content/gs-writing-walkthroughs-proc.adoc

```diff
@@ -463,15 +462,15 @@ Other attributes that are available to {walkthrough-name} authors are:
 
 
 .Prerequisites
-* You have created a repo using the {walkthrough-name} template.
-* You have added the {walkthrough-name} repo to your cluster as described in the {publishing-walkthrough} {walkthrough-name}.
+* You have created a repository using the {walkthrough-name} template.
+* You have added the {walkthrough-name} repository to your cluster as described in the {publishing-walkthroughs} {walkthrough-name}.
 
 For example, to add a link  to the Openshift Console in the {walkthrough-name}:
 
 . Add another step to the numbered list in your Procedure:
 +
 ----
-. Click on link:{openshift-url}[Openshift Console]
+. Click on the link to the Openshift Console.
```

Comment:
Navigate to the OpenShift console
or
Click on the OpenShift Console link in the Solution explorer
?

Reply:
After talking with @finp, this actually ok in the context it is used.